### PR TITLE
docs(avatar): add a11y documentation

### PR DESCRIPTION
**Pull request by bitwiseor**
a11y documentation for the avatar component

### CODE REVIEW

**Review by nicolethoen — APPROVED** [review]

File: packages/v4/src/content/accessibility/avatar/avatar.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+id: Avatar
+section: components
+---
+
+An **avatar** is a visual used to represent a user. It may contain an image or a placeholder graphic. Typical usage is to represent the current user in the masthead.
+
+**Keyboard users** shouldn’t be able to interact with the avatar on its own. Thus, the user should not be able to focus on the avatar using **Tab** to move forward and **Tab + Shift** to move backwards through interactive elements.
+
+**Screen reader users** should be able to navigate to the avatar and it should have alternative text, since it’s an image element: `aria-label` is the most common choice.
```
Comment (jessiehuff) [inline packages/v4/src/content/accessibility/avatar/avatar.md:10]: Would this be `alt` instead? I think this line and the below recommendations are slightly confusing together/clash.

File: packages/v4/src/content/accessibility/avatar/avatar.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+If you’re combining an avatar with another component, make sure to check accessibility guidelines for that component as well.
+
+
+## To make avatar accessible:
+- You may use the `alt` React prop to provide alternative text for the avatar image.
+- If you are using an SVG element for the avatar, [learn how to create accessible SVGs](https://www.deque.com/blog/creating-accessible-svgs/) and reference the different SVG patterns. Screen reader accessibility for SVGs varies based on the pattern being used.
+
+The following props/attributes have been added for you or are customizable in PatternFly:
+| Reason used | React prop used to customize | HTML attribute | React component that it should be applied to | Which HTML element it appears on in markup |
```
Comment (jessiehuff) [inline packages/v4/src/content/accessibility/avatar/avatar.md:20]: Just need a space here to make the table appear

File: packages/v4/src/content/accessibility/avatar/avatar.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+| -- | -- | -- | -- | -- |
+| Provides an accessible description of the avatar as it uses an image instead of text. | `alt` | aria-label="[avatar label]" | Avatar | .pf-c-avatar |
```
Comment (jessiehuff) [inline packages/v4/src/content/accessibility/avatar/avatar.md:22]: Similar to the button PR, I'm curious on opinions on this section for all of our PRs.

**Comment by nicolethoen** [comment]
@bitwiseor @jessiehuff 
Jessie is right, the `alt` react prop updates the `` html attribute, not the `aria-label`

Also, Jessie, in general I really like the tables. If our react components have props that are intentionally built to allow users to customize accessibility, then they are good to highlight in this way. If the attributes are not react props, then we don't need to highlight that. They can always add additional html attributes to a component if they want to - there are practically infinite possibilities.

**Review by nicolethoen — COMMENTED** [review]
need to update table with correct html attribute for alt

**Review by nicolethoen — APPROVED** [review]

File: packages/v4/src/content/accessibility/avatar/avatar.md (hunk 3 above)
Comment (jessiehuff) [reply packages/v4/src/content/accessibility/avatar/avatar.md]: Just needs a quick table update like the others, and it's good! 😄

**Review by jessiehuff — APPROVED** [review]
LGTM!

**Pull request merged**